aws-environments/infrastructure/variables.tf

```
@@ -0,0 +1,5 @@
+variable "project-name" {
```

Up to you, but usually the community prefer using underscore
```suggestion
variable "project_name" {
```